internal/dvc_oauth/auth.go

```diff
@@ -25,7 +25,7 @@ func GetAuthToken(clientId, clientSecret string) (Auth0, error) {
 	res, _ := http.DefaultClient.Do(req)
 
 	defer res.Body.Close()
-	body, _ := ioutil.ReadAll(res.Body)
+	body, _ := io.ReadAll(res.Body)
```

Comment:
IDE gave a deprecation warning on this, documentation indicated that this was a clear 1:1 swap. See https://pkg.go.dev/io/ioutil#ReadAll

Reply:
yep - that's fine